### PR TITLE
Phone

### PR DESCRIPTION
**Pull request by Cindie1908**
Added ternary on form Display in order to hide logo if there is no text

### CODE REVIEW

File: back/routes/places.js [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+          callback(null, './uploads');
+      }
+  })
+},
+filename: function (req, file, callback) {
+  callback(null, file.fieldname + '-' + Date.now());
+}
+});
+
+router.post('/api/file',function(req,res){
```
Comment (rocambille) [inline back/routes/places.js:22]: /api/files

File: back/routes/places.js [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+  var upload = multer({ storage : storage}).single('userFile');
```
Comment (rocambille) [inline back/routes/places.js:23]: const or let ;)

File: back/routes/server.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+app.post('/uploaddufichier', upload.single('monfichier'), function (req, res, next) {
+  fs.rename(req.file.path, 'public/images/' + req.file.originalname, function(err){
+    if (err) {
+        res.send('problème durant le déplacement');
+    } else {
+        res.send('Fichier uploadé avec succès');
+    }
+  });
+})
```
Comment (rocambille) [inline back/routes/server.js:15]: still needed?

File: front/src/component/DisplayAdmin.js [hunk 4]
```diff
@@ -20,8 +22,9 @@ function DisplayAdmin(props){
       <p>Responsable : {props.camping.firstname} {props.camping.lastname}</p>
     </div>
     <div id="mailPres">
-      <img className="logo" src='https://zupimages.net/up/19/49/jijs.png' alt="logo email"></img>
-      <p>{props.camping.email_company}</p>
+      {props.camping.email===null? "":
```
Comment (rocambille) [inline front/src/component/DisplayAdmin.js:25]: try:

    {props.camping.email===null || logo email</img>}

Comment (rocambille) [reply front/src/component/DisplayAdmin.js:25]: check other ternary operators

File: front/src/component/FormAdmin.js [hunk 5]
```diff
@@ -182,13 +182,10 @@ class FormAdmin extends React.Component {
...
-              type='text'
-              name='photo'
-              id='photo'
-              onChange={this.change}
-              required
-            />
+              <form method="POST" enctype="multipart/form-data" action="upload">
+                <input type="file" name="monfichier"/>
+                <button> envoyer </button>
+              </form>
```
Comment (rocambille) [inline front/src/component/FormAdmin.js:188]: no form in a form

**Review by rocambille — CHANGES_REQUESTED** [review]

File: front/src/component/DisplayAdmin.js (hunk 4 above)
Comment (Cindie1908) [reply front/src/component/DisplayAdmin.js:25]: Corrected :)

File: front/src/component/FormAdmin.js (hunk 5 above)
Comment (Cindie1908) [reply front/src/component/FormAdmin.js:188]: corrected :)

**Pull request closed without merging**